Model/Adminhtml/Config/PaymentPlans.php

```diff
@@ -139,4 +156,21 @@ public function beforeSave()
 
         return parent::beforeSave();
     }
+
+    /**
+     * @param $plans
+     * @return array
+     */
+    private function correctMinMaxLimit($plans):array
```

Comment:
Can you add a description and type parameter properly (I would like to know `$plans` is array of what 😉 )

This method is a little bit dirty (I'am not fan of "correcting" or "fixing" value previously built (maybe we hab to rework this information before this step)...

I let you decide if we have to do this now or if this is the best you can do according our short delay to deliver this feature (discussion is open if you want to talk about that now)